// @harbour-enterprises/superdoc/dist/superdoc.cjs
"use strict";
Object.defineProperty(exports, Symbol.toStringTag, { value: "Module" });
const superEditor_es = require("./chunks/super-editor.es-BIhNSEos.cjs");
const superdoc = require("./chunks/index-BXEyK17K.cjs");
require("./chunks/vue-DWle4Cai.cjs");
require("./chunks/jszip-b7l8QkfH.cjs");
const blankDocx = require("./chunks/blank-docx-CPqX9RF5.cjs");
exports.AnnotatorHelpers = superEditor_es.AnnotatorHelpers;
exports.Editor = superEditor_es.Editor;
exports.Extensions = superEditor_es.Extensions;
exports.SectionHelpers = superEditor_es.SectionHelpers;
exports.SuperConverter = superEditor_es.SuperConverter;
exports.createZip = superEditor_es.createZip;
exports.fieldAnnotationHelpers = superEditor_es.index$1;
exports.getRichTextExtensions = superEditor_es.getRichTextExtensions;
exports.superEditorHelpers = superEditor_es.helpers;
exports.trackChangesHelpers = superEditor_es.index;
exports.DOCX = superdoc.DOCX;
exports.HTML = superdoc.HTML;
exports.PDF = superdoc.PDF;
exports.SuperDoc = superdoc.SuperDoc;
exports.compareVersions = superdoc.compareVersions;
exports.getFileObject = superdoc.getFileObject;
exports.BlankDOCX = blankDocx.BlankDOCX;
